README.md

```diff
@@ -185,8 +192,7 @@ ansible-playbook migrate-db.yml                                    \
 cd concent-deployment/concent-builder/
 ansible-playbook deploy.yml                                        \
     --extra-vars cluster=$cluster                                  \
-    --inventory  ../../concent-deployment-values/ansible_inventory \
-    --user       $user
+    --inventory  ../../inventory
```

Comment:
@bartoszbetka What about this change?

Reply:
After discussing this, the issue was that with `ansible_inventory` the playbook would not work when run from inside the `concent-builder` itself. It would try to use the external IP and fail. The solution is to add `--connection=local` when running from `concent-builder`.

But the fact that it fails when using the external IP is not the typical situation. It's a special case that happens only if you're trying to configure a server while running from that server. Let's keep the instructions generic, with `ansible_inventory`. We could at most add a note somewhere that `--connection=local` exists. For example we could replace this in one of the previous sections:

``` markdown
All the instructions below assume that you're using the remote server.
```

with

``` markdown
All the instructions below assume that you're using local playbooks to run build and deployment commands on the remote server.

Note that if you're running the playbooks themselves from within that server too, you need to add `--connection=local` to your `ansible-playbook` calls.
Otherwise Ansible will run its commands over SSH (rather than directly) using the public IP specified in `ansible_inventory`, which will likely fail because you're not supposed to have your private SSH key on the remote server you're connecting to.
```